cohortextractor/log_utils.py

```diff
@@ -0,0 +1,63 @@
+import logging.config
```

Comment:
copied over from V1

Reply:
I can't remember who wrote this, though I'm pretty sure it wasn't me. It all looks pretty sensible though so I don't see a problem with copying it verbatim for now.